src/common/hooks/usePiecesExpect/usePiecesExpect.js

```diff
@@ -0,0 +1,14 @@
+import { useMutation } from 'react-query';
+
+export const usePiecesExpect = () => {
+  const { isLoading, mutateAsync } = useMutation({
+    mutationFn: (_pieces) => {
+      return Promise.reject(new Error('TODO: Not implemented yet'));
```

Comment:
When is it planned to be implemented? I suggest following YAGNI principle here if you are not 100% sure that this will be needed.

Reply:
The ticket for connecting BE and UI - [UIREC-307](https://issues.folio.org/browse/UIREC-307) - lays in the next sprint. I've updated a message to put a reference for that ticket.